docs/docs/configure-task-handling.md

```diff
@@ -0,0 +1,18 @@
+---
+Configuring Task Handling
+---
+
+You can configure Marathon's actions on unreachable tasks. The `unreachableStrategy` parameter of your app definition allows you to configure this in two ways: by defining when a new task instance should be launched, and by defining when a task instance should be expunged.
```

Comment:
It looks like `killSelection` is in the top level of the app definition and the `PodSchedulingPolicy` level of the pod definition. I'm not sure how much more documentation is needed about this apart from the description you've written. If more _would_ be useful, where would be the best place for it? The "Blue-Green Deployment" section, perhaps?